reader-shp/src/main/java/com/graphhopper/reader/shp/OSMShapeFileReader.java

```diff
@@ -246,6 +246,12 @@ public DataReader setWayPointMaxDistance(double wayPointMaxDistance) {
         return this;
     }
 
+    @Override
+    public DataReader setSmoothElevation(boolean smoothElevation) {
+        // TODO implement elevation smoothing for shape files
+        return this;
+    }
```

Comment:
Do we need elevation smoothing for shape files?

Reply:
Can be done later